schema_based_on_diagram.sql

```diff
@@ -0,0 +1,53 @@
+CREATE TABLE medical_histories (
```

Comment:
- [ ] Kindly add indexes for all foreign keys as per this [requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/database_from_diagram.md#:~:text=Remember%20to%20add%20the%20FK%20indexes.).

Reply:
Thanks I just did that